docs/dev/compiler-toolchain/llvm.md

```diff
@@ -0,0 +1,22 @@
+# LLVM Framework
+
+[The LLVM framework](https://github.com/matter-labs/era-compiler-llvm) is an excellent choice for developing toolchains for smart contract languages. Its powerful intermediate representation (IR) allows developers to design, implement, and optimize efficient
```

Comment:
“an excellent choice” sounds like empty bragging.  Perhaps (echoing Lattner’s article and one of our job postings) “a soundly-architected and well-tested framework”?

Reply:
Indeed sounds more marketing-ish and doesn't match the style of the rest of the section.

Reply:
9b9b5715